tests/contrib/io/azure/test_csv_blob.py

```diff
@@ -338,15 +338,13 @@ def test_save_blob_args(blob_from_mock, blob_csv_data_set, dummy_dataframe):
     )
 
 
-# pylint: disable=protected-access
 def test_load_extra_params(blob_csv_data_set):
     expected_keys = ["option"]
     data_set = blob_csv_data_set(load_args={"option": "value"})
     for expected_key in expected_keys:
         assert expected_key in data_set._load_args.keys()
 
 
-# pylint: disable=protected-access
```

Comment:
I'm surprised this doesn't complain again.

Reply:
It has been globally disabled for everything in `tests/` in `.pre-commit-config.yaml`